azure/services/managedclusters/spec_test.go

```diff
@@ -69,7 +69,7 @@ func TestParameters(t *testing.T) {
 					"test-tag": "test-value",
 				},
 				Version:         "v1.22.0",
-				LoadBalancerSKU: "Standard",
+				LoadBalancerSKU: "standard",
```

Comment:
Do we need to do some conversion ourselves so as not to break existing users with clusters that already specify (capital) `Standard` or `Basic`? Or is this not actually case-sensitive? I see none of our CI templates specify anything for this field.

Reply:
AFAICT in reality the API accepts either `Standard` or `standard` (and so on), but the API now prefers all lower-case in its constants and tests.